vertical-pod-autoscaler/pkg/recommender/logic/recommender.go

```diff
@@ -99,13 +107,19 @@ func FilterControlledResources(estimation model.Resources, controlledResources [
 
 // CreatePodResourceRecommender returns the primary recommender.
 func CreatePodResourceRecommender() PodResourceRecommender {
-	targetCPUPercentile := 0.9
-	lowerBoundCPUPercentile := 0.5
-	upperBoundCPUPercentile := 0.95
+	targetCPUPercentile := *targetCPUPercentile
+	lowerBoundCPUPercentile := *lowerBoundCPUPercentile
+	upperBoundCPUPercentile := *upperBoundCPUPercentile
 
-	targetMemoryPeaksPercentile := 0.9
-	lowerBoundMemoryPeaksPercentile := 0.5
-	upperBoundMemoryPeaksPercentile := 0.95
+	if lowerBoundCPUPercentile <= 0 || lowerBoundCPUPercentile > targetCPUPercentile || targetCPUPercentile > upperBoundCPUPercentile {
+		klog.Fatalf("Please make sure lowerBoundCPUPercentile > 0 and lowerBoundCPUPercentile <= targetCPUPercentile <= upperBoundCPUPercentile")
+	}
+	targetMemoryPeaksPercentile := *targetMemoryPercentile
+	lowerBoundMemoryPeaksPercentile := *lowerBoundMemoryPercentile
+	upperBoundMemoryPeaksPercentile := *upperBoundMemoryPercentile
+	if lowerBoundMemoryPeaksPercentile <= 0 || lowerBoundMemoryPeaksPercentile > targetMemoryPeaksPercentile || targetMemoryPeaksPercentile > upperBoundMemoryPeaksPercentile {
+		klog.Fatalf("Please make sure lowerBoundMemoryPercentile > 0 and lowerBoundMemoryPercentile <= targetMemoryPercentile <= upperBoundMemoryPercentile")
```

Comment:
Can you add this information in flag descriptions? For example
lowerBoundCPUPercentile    = flag.Float64("recommendation-lower-bound-cpu-percentile", 0.5, `Percentile of cpu usage histogram for lower bound recommendation for a container. Needs to be lower than recommendation-target-cpu-percentile`)